Tests/ArgumentParserInteractiveTests/ArgumentInteractiveTests.swift

```diff
@@ -0,0 +1,82 @@
+//===----------------------------------------------------------*- swift -*-===//
+//
+// This source file is part of the Swift Argument Parser open source project
+//
+// Copyright (c) 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+import ArgumentParser
```

Comment:
If you change this to a `@testable import`, you can make a version of the `AssertParseCommand` function that creates a `CommandParser` directly, instead of going through public API.